### ISSUE TITLE
In Windows a iDynTree.Visualizer does now re-open a window after a viz.init()/viz.close()/viz.init() cycle with sdl 1.2.52, works with sdl 1.2.15

### ISSUE THREAD

**Issue by traversaro** [robotology/idyntree#986]
See https://github.com/robotology/idyntree/issues/985#issuecomment-1113576115 . 

It can be reproduced as well in Python:
~~~
import idyntree.bindings
viz = idyntree.bindings.Visualizer()
viz.init()
viz.draw()
viz.close()
viz.init()
viz.draw()
viz.close()
~~~

**Comment by traversaro** [comment]
I played a bit with the problem in Python, and the output of `viz.init` is quite interesting. The first time the output is:
~~~
>>> viz = idyntree.bindings.Visualizer()
>>> viz.init()
Irrlicht Engine version 1.8.5
SDL Version 1.2.15
Using renderer: OpenGL 4.6.0
Intel(R) UHD Graphics: Intel
OpenGL driver version is 1.2 or better.
GLSL version: 4.6
True
>>> viz.run()
True
>>> viz.close()
>>> viz.init()
Irrlicht Engine version 1.8.5
Unable to initialize SDL!: directx not available
SDL Version 1.2.15
Using renderer: OpenGL
OpenGL driver version is not 1.2 or better.
Failed to load OpenGL's multitexture extension, proceeding without.
Warning: OpenGL device only has one texture unit. Disabling multitexturing.
GLSL not available.
True
>>> viz.run()
False
>>> viz.close()
~~~ 

The first time everything starts correctly, while the second time for some reason something related to `directx` is failing. Why this difference? I inspected the irrlicht's SDL device source code, and indeed it seems that **after** SDL_init is called the first time, irrlicht sets the `SDL_VIDEODRIVER` env variable to `directx` : 

https://github.com/zaki/irrlicht/blob/97472da9c22ae4a49dfcefb9da5156601fa6a82a/source/Irrlicht/CIrrDeviceSDL.cpp#L81

That variable is set **after** the first `SDL_init`  is called so it does not influence it, but the environment variables are something that remain persistent at the process level, so any future call to `SDL_init`  (even if we create a new `iDynTree::Visualizer` object) will fail due to the fact that `SDL_VIDEODRIVER` is set to `directx` . The ideal solution would be that irrlicht does not interact with global properties of the process such as environment variable, but a possible workaround on the iDynTree side is just to unset that environment variable irrlicht is launched.
